packages/oc-template-react-compiler/__tests__/__snapshots__/compileServer.js.snap

```diff
@@ -2,10 +2,10 @@
 
 exports[`Should correctly compile the server 1`] = `
 Object {
-  "hashKey": "0c3ff55fb661438c887173d4454b1b3ddf745374",
+  "hashKey": "a1d50b21a0282a39515de9b50a333f9b4ab8d9c7",
   "src": "server.js",
   "type": "node.js",
 }
 `;
 
-exports[`Should correctly compile the server 2`] = `"module.exports=function(a){function b(d){if(c[d])return c[d].exports;var e=c[d]={i:d,l:!1,exports:{}};return a[d].call(e.exports,e,e.exports,b),e.l=!0,e.exports}var c={};return b.m=a,b.c=c,b.d=function(a,c,d){b.o(a,c)||Object.defineProperty(a,c,{configurable:!1,enumerable:!0,get:d})},b.n=function(a){var c=a&&a.__esModule?function(){return a['default']}:function(){return a};return b.d(c,'a',c),c},b.o=function(a,b){return Object.prototype.hasOwnProperty.call(a,b)},b.p='',b(b.s=0)}([function(a,b,c){'use strict';Object.defineProperty(b,'__esModule',{value:!0});var d=c(1);b.data=(a,b)=>{Object(d.a)(a,(c,d)=>{if(c)return b(c);const e=Object.assign({},d,{_staticPath:a.staticPath,_baseUrl:a.baseUrl,_componentName:'react-component',_componentVersion:'1.0.0'}),f=a.env&&'local'===a.env.name?a.staticPath:'https:'+a.staticPath;return b(null,Object.assign({},{reactComponent:{key:'666',src:f+'react-component.js',props:e}}))})}},function(a,b){'use strict';b.a=(a,b)=>{const c=a.params.name;return b(null,{name:c})}}]);"`;
+exports[`Should correctly compile the server 2`] = `"module.exports=function(a){function b(d){if(c[d])return c[d].exports;var e=c[d]={i:d,l:!1,exports:{}};return a[d].call(e.exports,e,e.exports,b),e.l=!0,e.exports}var c={};return b.m=a,b.c=c,b.d=function(a,c,d){b.o(a,c)||Object.defineProperty(a,c,{configurable:!1,enumerable:!0,get:d})},b.n=function(a){var c=a&&a.__esModule?function(){return a['default']}:function(){return a};return b.d(c,'a',c),c},b.o=function(a,b){return Object.prototype.hasOwnProperty.call(a,b)},b.p='',b(b.s=0)}([function(a,b,c){'use strict';Object.defineProperty(b,'__esModule',{value:!0});var d=c(1);b.data=(a,b)=>{Object(d.a)(a,(c,d)=>{if(c)return b(c);const e=Object.assign({},d,{_staticPath:a.staticPath,_baseUrl:a.baseUrl,_componentName:'react-component',_componentVersion:'1.0.0'}),f=a.env&&'local'===a.env.name?a.staticPath:'https:'+a.staticPath;return b(null,Object.assign({},{reactComponent:{key:'9226c0643c9b97f8cbe8fcfd3cf8d343b8d88eba',src:f+'react-component.js',props:e}}))})}},function(a,b){'use strict';b.a=(a,b)=>{const c=a.params.name;return b(null,{name:c})}}]);"`;
```

Comment:
We have an issue here, as the update od the acceptance seems to alter the unit as well, I think we have some unit that is not that unit 😅 ...